src/EventRegistry.php

```diff
@@ -160,4 +161,13 @@ public function getEventClassName(string $event)
 
         return null;
     }
+
+    public function listActiveEvents(array $events)
+    {
+        foreach ($events as &$event) {
+            $event .= sprintf(' (<fg=yellow>%s</>)', self::$eventsMap[$event]);
+        }
+
+        return $events;
```

Comment:
Can you describe what this is doing exactly? I have a hard time just reading the code to figure it out :)

Reply:
It generates more verbose event describing
<img width="771" alt="Screen Shot 2019-12-16 at 16 23 34" src="https://user-images.githubusercontent.com/213810/70914610-fe8e4380-2020-11ea-8f59-37d9c97ef3da.png">

Reply:
Also, I restored old-style event keys like `kernel.request` because it's still described in symfony5 before it command asked the user to enter long FQCN for `RequestEvent` class.